app/views/layouts/application.html.slim

```diff
@@ -12,7 +12,8 @@ html lang="#{I18n.locale}"
       = content_for?(:title) ? APP_NAME + ' - ' + yield(:title) : APP_NAME
 
     == stylesheet_link_tag 'application', media: 'all'
-    == javascript_include_tag 'application'
+    - unless controller_name == 'home'
+      == javascript_include_tag 'application'
```

Comment:
Agreed it's a little icky, but i thought the benefit was worth it (and it follows a pattern I saw in another view - https://github.com/18F/identity-idp/blob/master/app/views/devise/shared/_links.html.slim#L1)